app/components/application_record_card/view.rb

```diff
@@ -29,8 +31,22 @@ def route
       record.record_type.humanize
     end
 
+    def updated_at
+      date_stamp = record.updated_at.presence || record.created_at
+      date_text = tag.span(date_stamp.strftime("%-d %B %Y"))
+      class_list = "govuk-caption-m govuk-!-font-size-16 govuk-!-margin-top-2 govuk-!-margin-bottom-0 app-application-card__submitted"
+
+      sanitize(tag.p(date_text.prepend("Updated: "), class: class_list))
+    end
+
     def status
       "Draft"
     end
+
+    def trainee_id
```

Comment:
Could we add a postfix to these sort of html methods? I think the rails convention is "_tag". So we would end up with `trainee_id_tag`.